curriculum/challenges/english/25-front-end-development/quiz-javascript-functions/66edcc779993c0da6906dbb9.md

```diff
@@ -17,439 +17,439 @@ Answer all of the questions below correctly to pass the quiz.
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+Which of the following is an example of a method?
 
 #### --distractors--
 
-Placeholder distractor 1
+`function greet(name) { console.log("Hello, " + name); }`
 
 ---
 
-Placeholder distractor 2
+`var greet = function(name) { console.log("Hello, " + name); };`
 
 ---
 
-Placeholder distractor 3
+All of the above.
 
 #### --answer--
 
-Placeholder answer
+`const person = { name: "Alice", greet: function() { console.log("Hello, my name is " + this.name); } };`
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+What is the benefit of using methods instead of functions?
 
 #### --distractors--
 
-Placeholder distractor 1
+Methods provide a more organized way to group related functionality.
 
 ---
 
-Placeholder distractor 2
+Methods can be accessed using dot notation, which can make code more readable.
 
 ---
 
-Placeholder distractor 3
+Methods have access to the properties and methods of their object.
 
 #### --answer--
 
-Placeholder answer
+All of the above.
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+What is the difference between arguments and parameters?
 
 #### --distractors--
 
-Placeholder distractor 1
+There is no difference between arguments and parameters.
 
 ---
 
-Placeholder distractor 2
+Parameters are the actual values passed to a function when it is called, while arguments are the names of variables used in a function definition.
 
 ---
 
-Placeholder distractor 3
+Arguments and parameters are used interchangeably.
 
 #### --answer--
 
-Placeholder answer
+Parameters are the names of variables used in a function definition, while arguments are the actual values passed to the function when it is called.
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+What happens if a function is called with more arguments than it expects?
 
 #### --distractors--
 
-Placeholder distractor 1
+The extra arguments will be assigned to the remaining parameters.
 
 ---
 
-Placeholder distractor 2
+The function will throw an error.
 
 ---
 
-Placeholder distractor 3
+The function will use the default values specified in the function definition.
 
 #### --answer--
 
-Placeholder answer
+The extra arguments will be ignored.
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+What is the correct syntax for defining a function with default parameter values?
 
 #### --distractors--
 
-Placeholder distractor 1
+`function myFunction(param1 = value1, param2) { ... }`
 
 ---
 
-Placeholder distractor 2
+`function myFunction(param1, param2 = value2) { ... }`
 
 ---
 
-Placeholder distractor 3
+`function myFunction(param1: value1, param2: value2) { ... }`
 
 #### --answer--
 
-Placeholder answer
+`function myFunction(param1 = value1, param2 = value2) { ... }`
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+What is the default value for a missing argument in a function call?
 
 #### --distractors--
 
-Placeholder distractor 1
+Error
```

Comment:
```suggestion
`Error`
```

Reply:
and the distractors and answer for this question need to be wrapped in backticks